cluster-autoscaler/simulator/clustersnapshot/store/basic.go

```diff
@@ -213,6 +215,16 @@ func (snapshot *BasicSnapshotStore) DraSnapshot() *drasnapshot.Snapshot {
 	return snapshot.draSnapshot
 }
 
+// CsiSnapshot returns the CSI snapshot.
+func (snapshot *BasicSnapshotStore) CsiSnapshot() *csisnapshot.Snapshot {
+	return snapshot.csiSnapshot
+}
+
+// CSINodes returns the CSI nodes snapshot.
+func (snapshot *BasicSnapshotStore) CSINodes() fwk.CSINodeLister {
```

Comment:
Could you move this method to where the other scheduler-framework-specific methods are (after `DeviceClassResolver()`)?